### ISSUE TITLE
Validation loss not showing and the plot takes it as a training loss

### ISSUE THREAD

**Issue by BankNatchapol** [JarodMica/ai-voice-cloning#36]
The spike in this graph is actually validation losses.

![output-onlinepngtools (2)](https://github.com/JarodMica/ai-voice-cloning/assets/53829777/6bb99837-87c3-4882-9677-0cef173fac4b)

just add 
`data["mode"] = mode`
in the src/utils.py -> load_statistics 

and it's fixed

![output-onlinepngtools (3)](https://github.com/JarodMica/ai-voice-cloning/assets/53829777/6d440495-9b5b-4cca-bf1b-bf0a106d712d)
